src/parseTools.mjs

```diff
@@ -921,7 +922,12 @@ function makeModuleReceiveWithVar(localName, moduleName, defaultValue) {
 function makeRemovedFSAssert(fsName) {
   assert(ASSERTIONS);
   const lower = fsName.toLowerCase();
-  if (JS_LIBRARIES.includes(path.resolve(path.join('lib', `lib${lower}.js`)))) return '';
+  if (
+    JS_LIBRARIES.includes(
+      path.join(path.dirname(fileURLToPath(import.meta.url)), 'lib', `lib${lower}.js`),
```

Comment:
I think you can do `localFile(path.join('lib', `lib${lower}.js`))` perhaps.

Can we add a test case maybe, to be sure its fixed?